src/locale/languages/br.js

```diff
@@ -0,0 +1,330 @@
+      new: 'NOVO',
+      deleted: 'REMOVIDO',
+      fileRenamed: 'Arquivo renomeado sem qualquer alteração',
+    },
+  },
+  organization: {
+    main: {
+      membersTitle: 'MEMBROS',
+      descriptionTitle: 'DESCRIÇÃO',
+    },
+  },
+  issue: {
+    settings: {
+      title: 'Configurações',
+      pullRequestType: 'Pull Request',
+      issueType: 'Issue',
+      applyLabelButton: 'Aplicar Label',
+      noneMessage: 'Nenhuma ainda',
+      labelsTitle: 'LABELS',
+      assignYourselfButton: 'Atribua a você',
+      assigneesTitle: 'ATRIBUÍDA A',
+      actionsTitle: 'AÇÕES',
+      unlockIssue: 'Destrancar {{issueType}}',
+      lockIssue: 'Trancar {{issueType}}',
+      closeIssue: 'Fechar {{issueType}}',
+      reopenIssue: 'Reabrir {{issueType}}',
+      areYouSurePrompt: 'Tem certeza?',
+      applyLabelTitle: 'Aplicar um label a esta issue',
+    },
+    main: {
+      assignees: 'Atribuída a',
+      mergeButton: 'Merge Pull Request',
+      noDescription: 'Nenhuma descrição fornecida.',
+      lockedCommentInput: 'Trancada, mas você ainda pode comentar...',
+      commentInput: 'Adicionar um comentário...',
+      commentButton: 'Postar',
+      lockedIssue: 'A issue está trancada',
+      states: {
+        open: 'Aberta',
+        closed: 'Fechada',
+        merged: 'Merged',
+      },
+      screenTitles: {
+        issue: 'Issue',
+        pullRequest: 'Pull Request',
+      },
+    },
+    newIssue: {
+      title: 'Nova Issue',
+      missingTitleAlert: 'A issue precisa ter um título!',
+      issueTitle: 'Título da Issue',
+      writeATitle: 'Escreva um título para sua issue aqui',
+      issueComment: 'Comentário da Issue',
+      writeAComment: 'Escreva um comentário para sua issue aqui',
+    },
+    pullMerge: {
+      title: 'Merge Pull Request',
+      createMergeCommit: 'Criar um merge commit',
+      squashAndMerge: 'Squash e criar merge',
+      merge: 'merge',
+      squash: 'squash',
+      missingTitleAlert: 'O commit precisa ter um título!',
+      commitTitle: 'Título do Commit',
+      writeATitle: 'Escreva um título para seu commit aqui',
+      commitMessage: 'Messagem do Commit',
+      writeAMessage: 'Escreva uma mensagem para seu commit aqui',
+      mergeType: 'Tipo de Merge',
+      changeMergeType: 'Trocar Tipo de Merge',
+    },
+  },
+  common: {
+    bio: 'BIO',
+    stars: 'Favoritos',
+    orgs: 'ORGANIZAÇÕES',
+    noOrgsMessage: 'Nenhuma organização',
+    info: 'INFO',
+    company: 'Empresa',
+    location: 'Localização',
+    email: 'E-mail',
+    website: 'Website',
+    repositories: 'Repositórios',
+    followers: 'Seguidores',
+    following: 'Seguindo',
+    cancel: 'Cancelar',
+    yes: 'Sim',
+    ok: 'OK',
+    submit: 'Enviar',
+    relativeTime: {
+      past: '%s',
+      s: '%ds',
+      m: '%dm',
+      mm: '%dm',
+      h: '%dh',
+      hh: '%dh',
+      d: '%dd',
+      dd: '%dd',
+      M: '%dmo',
+      MM: '%dmo',
+      y: '%dy',
+      yy: '%dy',
+    },
+  },
+};
```

Comment:
I'm not a language expert by any means. How different are Portuguese and Brazilian Portuguese? If they're very similar (UK English vs. US English), I think it probably makes more sense just to use "Portuguese".

Reply:
In my opinion, Portugal and Brazilian Portuguese are not very similar like UK and US English, altough they share a lot of aspects, but vocabulary, phonetics and syntax are differ a bit. But yeah, maybe we could use the same translation.

Reply:
Alright, if they're not too similar then this is totally fine!